documentation/autoinstall.md

```diff
@@ -2,7 +2,9 @@
 
 ## Introduction
 
-The server installer for 20.04 supports a new mode of operation: automated installation, autoinstallation for short. You might also know this feature as unattended or handsoff or preseeded installation.
+Since version 20.04, the server installer supports the automated installation
```

Comment:
Is Discourse smart enough to flow these correctly like it was one long line?  I think the answer is no, it would be nice to 80 column these files but I don't think we can without a post-processing step.

Reply:
Thanks! Looks like you're right .. Well I'll discard the whitespace changes for now then.